git-branchless-opts/src/lib.rs

```diff
@@ -396,6 +396,10 @@ pub struct SubmitArgs {
     /// An optional message to include with the create or update operation.
     #[clap(short = 'm', long = "message")]
     pub message: Option<String>,
+
+    /// Fetch remote but don't push or create anything; only show what would be pushed or created.
+    #[clap(short = 'n', long = "dry-run")]
+    pub dry_run: bool,
```

Comment:
I have exactly 0 invested in these flags and would welcome any input on what to use. I seem to recall other tools dry run flags being `-n/--dry-run` but I'm open to anything.

Reply:
We use `-n`/`--dry-run` in one other place, so this is fine.

https://github.com/arxanas/git-branchless/blob/5871499a63f8332e3780554baa9b256d6225dc6e/git-branchless-opts/src/lib.rs#L892

(But... in `scm-record`, we use `-N`/`--dry-run` instead 😬).

Reply:
Oh, of course, I didn't even think to look around in this project. I've updated the rustdoc comment here to be a little more similar to the other `--dry-run` in this file.